### DISCUSSION TITLE
run the desktop instasnce on startup without gui linux?

### DISCUSSION THREAD

**Discussion by payrim** [TriliumNext/Trilium#2846] [Q&A]
why opening trilium desktop for the second time opens in an instant, but after reboot it takes more time?

i checked the processes and i'm sure that trilium internal server (and there's app,asar) closes upon closing the desktop instance but still it opens way faster the second time it gets opened.

what causes this? does the internal server or database somehow still run on background? and how can i run the instance without gui at startup in the same way so that i could open it faster when i needed it?

**Comment by zadam** [comment]
Hi, I suspect there's some OS (filesystem) caching or something similar, there's nothing special done. When the GUI is closed, the whole process is ended.

> and how can i run the instance without gui at startup in the same way so that i could open it faster when i needed it?

Seeing you run on Linux, there should be a way to let it start up with a minimized window, depends on the window manager I guess.
